logstash-filter-grok.gemspec

```diff
@@ -22,10 +22,11 @@ Gem::Specification.new do |s|
   # Gem dependencies
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
   s.add_runtime_dependency "logstash-core", ">= 5.6.0"
+  #s.add_runtime_dependency 'logstash-mixin-ecs_compatibility_support', '~> 1.0'
 
   s.add_runtime_dependency 'jls-grok', '~> 0.11.3'
   s.add_runtime_dependency 'stud', '~> 0.0.22'
-  s.add_runtime_dependency 'logstash-patterns-core'
+  s.add_runtime_dependency 'logstash-patterns-core' #, '>= 5.0' TODO
```

Comment:
We should be fine with a minor release of `logstash-patterns-core` that additively provides the selector and continues to resolve legacy when no selector is explicitly provided.

Reply:
yeah, all will be backwards compatible. initially started with an assumption that it won't be ...

Reply:
still have doubt about this - bumping patterns-core from (current) 4.2.0 -> 5.0.0 could make our intent clearer.
the filter can "clearly" declare: `s.add_runtime_dependency 'logstash-patterns-core', '>= 5.0.0', '< 6'`,
- filter requires a new feature provided by patterns-core in order to work properly (with `ecs_compatibility => v1`)
- no limit previously `s.add_runtime_dependency 'logstash-patterns-core'` clearer to distinguish with a >= major
- there's room for releasing another patterns-core minor 4.x for existing versions as/if needed
- patch updates to already releaed LS 7.x versions won't pull in patterns-core 5.0.0
- minor LS 7.x gem updates will pull 5.0.0 in as long as they update the grok filter minor version